tools/quota-monitoring-alerting/python/config.yaml

```diff
@@ -0,0 +1,35 @@
+# Configuration data for building Quota dashboards and pushing custom
```

Comment:
All Source files should have a License Header. Please add one

Reply:
Done